scene/gui/color_picker.cpp

```diff
@@ -1530,7 +1530,7 @@ void ColorPicker::_bind_methods() {
 }
 
 ColorPicker::ColorPicker() :
-		BoxContainer(true) {
+		VBoxContainer() {
```

Comment:
Isn't this initialization unnecessary now?

Reply:
Yep, that's right.  I'll get rid of it in my next push.